multi-int-store/src/main/java/com/connexta/multiintstore/repositories/IndexedMetadataRepository.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.multiintstore.repositories;
+
+import com.connexta.multiintstore.models.IndexedProductMetadata;
+import java.util.List;
+import org.springframework.data.solr.repository.SolrCrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface IndexedMetadataRepository
+    extends SolrCrudRepository<IndexedProductMetadata, String> {
+
+  List<IndexedProductMetadata> findByContents(String keyword);
```

Comment:
❓ Is this method built in and already handles the query for us? I'm a little confused by this syntax.

Reply:
The magic of Spring data does the... magic. It will look for a Solr field named "contents". 
https://docs.spring.io/spring-data/data-solr/docs/1.0.0.RC1/reference/htmlsingle/#solr.query-methods.criterions